app/models/manageiq/providers/redhat/infra_manager/ovirt_services.rb

```diff
@@ -693,6 +703,12 @@ def collect_external_network_providers
 
     private
 
+    def select_graphics_console(consoles, pref_type = 'spice')
+      preferred = consoles.find { |c| c.protocol.downcase == pref_type }
+
```

Comment:
Perhaps we should add a log message here in case a console of the preferred type was not found?

Reply:
do we ever use pref_type?
and should we test it?

Reply:
yeah you're right, I'll add some logs. Just wondering how to get logger reference in a simple way.

Reply:
No right know we're not using pref_type directly as a parameter. I afded it 'cause I supposed the preferred one is SPICE but leaving a way in the private method call to express another option. Not sure if to leave it as parameter or change it in a const...